.gitignore

```diff
@@ -2,4 +2,4 @@ _site
 .DS_Store
 *.swp
 .sass-cache
-
+.idea
```

Comment:
Ja, kann man machen. Ich wundere mich bei manchen dieser Ignores, warum ihr die nicht in euren globalen Git-Settings habt? 😄